src/process-events.c

```diff
@@ -622,11 +620,13 @@ static __always_inline int exit_exec(struct pt_regs *__ctx, u32 i_rdev, u64 i_in
 
     file_info_t fi = {
         .inode = i_ino,
-        .devmajor = MAJOR(i_rdev),
-        .devminor = MINOR(i_rdev),
-        .value[0] = '\0',
+        .devmajor = MAJOR(i_dev),
+        .devminor = MINOR(i_dev),
+        .comm = {0},
```

Comment:
Setting `.comm[0] = '\0'` and manually ensuring that it's null-terminated _may_ end up with less instructions than `.comm = {0}`. I think, by default, clang will fill this up with instructions to zero out the field byte-by-byte, which may not be what you want. If the `get_current_comm` helper handles null termination for you, then you may not need to initialize this at all.

Reply:
The `bpf_get_current_comm` hasn't always handled NULL termination but it does in 4.11 and on so I think we should be fine